src/Components/Restaurant.js

```diff
@@ -1,15 +1,18 @@
 import React from "react";
+import Order from "./Restaurant/Order";
 
 const Restaurant = () => {
-  const pizzas = 0;
   return (
-    <div>
-      <h3>Restaurant Orders</h3>
-      <ul>
-        <li>
-          Pizzas: {pizzas} <button className="btn btn-primary">Add</button>
-        </li>
-      </ul>
+    <div className="container">
+      <div className="restaurant">
+        <h4>Restaurant Orders</h4>
+      </div>
+      <div className="pizzas">
+        <ul>
+          <Order orderType="Pizzas:" />
```

Comment:
A small point, but do you think the colon (`:`) should be in the prop? I'd say that is part of the *presentation* of the `Order` component, so the `Restaurant` component shouldn't have to worry about it.

Another way of thinking about this: what would happen if I asked you to show the orders in a different way, without the colon? What components would you have to change to do that?

Reply:
Order component.